themes/bootstrap3/templates/admin/tags/home.phtml

```diff
@@ -6,17 +6,16 @@
   <div class="<?=$this->layoutClass('mainbody')?>">
     <h2><?=$this->translate('Tag Management')?></h2>
 
-    <?=$this->render("admin/tags/menu.phtml")?>
-
-    <h3><?=$this->translate('Statistics')?></h3>
-    <table class="table table-striped">
-      <tr><th><?=$this->transEsc('total_users')?></th><th><?=$this->transEsc('total_resources')?></th><th><?=$this->transEsc('total_tags')?></th><th><?=$this->transEsc('unique_tags')?></th><th><?=$this->transEsc('anonymous_tags')?></th></tr>
-      <tr><td><?=$statistics['users']?></td><td><?=$statistics['resources']?></td><td><?=$statistics['total']?></td><td><?=$statistics['unique']?></td><td><?=$statistics['anonymous']?></td></tr>
-    </table>
+    <div class="toolbar">
```

Comment:
I'm not sure I understand how these changes fit in or what they are supposed to be doing. I think this may be removing more stuff that is related to social stats and does not actually need to go.